### DISCUSSION TITLE
Dumb question.

### DISCUSSION THREAD

**Discussion by CixsCR2** [CappielloAntonio/tempo#235] [Q&A]
Hi, new user of tempo loving the app so far. Quick question, what settings should I use for lower filesizes on downloads? I wish to keep a copy of my playlist on my phone. (also, where does tempo store music files for downloaded music?)

Thanks in advance!

**Comment by Neurology0443** [comment]
> also, where does tempo store music files for downloaded music?

You can't directly access the downloaded music. #153
